docs/database/use-entity-framework-db-contexts.md

```diff
@@ -0,0 +1,564 @@
+
+:::zone pivot="sql-server-ef"
+
+```csharp
+builder.Services.AddDbContextFactory<ExampleDbContext>(options =>
+    options.UseSqlServer(builder.Configuration.GetConnectionString("database")
+        ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+
+builder.EnrichSqlServerDbContext<ExampleDbContext>(
+    configureSettings: settings =>
+    {
+        settings.DisableRetry = false;
+        settings.CommandTimeout = 30 // seconds
+    });
+```
+
+:::zone-end
+:::zone pivot="postgresql-ef"
+
+```csharp
+builder.Services.AddDbContextFactory<ExampleDbContext>(options =>
+    options.UseNpgsql(connectionString
+        ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+
+builder.EnrichNpgsqlDbContext<ExampleDbContext>(
+    configureSettings: settings =>
+    {
+        settings.DisableRetry = false;
+        settings.CommandTimeout = 30 // seconds
+    });
+```
+
+:::zone-end
+:::zone pivot="oracle-ef"
+
+```csharp
+builder.Services.AddDbContextFactory<ExampleDbContext>(options =>
+    options.UseOracle(connectionString
+        ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+
+builder.EnrichOracleDatabaseDbContext<ExampleDbContext>(
+    configureSettings: settings =>
+    {
+        settings.DisableRetry = false;
+        settings.CommandTimeout = 30 // seconds
+    });
+```
+
+:::zone-end
+:::zone pivot="mysql-ef"
+
+```csharp
+builder.Services.AddDbContextFactory<ExampleDbContext>(options =>
+    options.UseMySql(connectionString
+        ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+
+builder.EnrichMySqlDbContext<ExampleDbContext>(
+    configureSettings: settings =>
+    {
+        settings.DisableRetry = false;
+        settings.CommandTimeout = 30 // seconds
+    });
+```
+
+:::zone-end
+
+Notice that the above code adds and enriches a *database context factory* in the DI container. When you retreive this from the container, you must add a line of code to create a *database context* from it:
+
+```csharp
+public class ExampleService(IDbContextFactory<ExampleDbContext> contextFactory)
+{
+    using (var context = contextFactory.CreateDbContext())
+    {
+        // Use context...
+    }
+}
+```
+
+Database contexts created from factories in this way aren't disposed of automatically because they aren't tied to an HTTP request lifetime. You must make sure your code disposes of them. In this example, the `using` code block ensures the disposal.
```

Comment:
```suggestion
Contexts created from factories in this way aren't disposed of automatically because they aren't tied to an HTTP request lifetime. You must make sure your code disposes of them. In this example, the `using` code block ensures the disposal.
```

Reply:
Fixed.